tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/Responses/CLICheckResponse.cs

```diff
@@ -0,0 +1,67 @@
+using System.Text.Json.Serialization;
+
+namespace Azure.Sdk.Tools.Cli.Models;
+
+/// <summary>
+/// Base class for CLI check responses with exit code and output.
+/// </summary>
+public class CLICheckResponse: Response
+{
+    [JsonPropertyName("exit_code")]
+    public int ExitCode { get; set;}
+    
+    [JsonPropertyName("output")]
+    public string Output { get; set;}
```

Comment:
nit: Please consider renaming this to CheckStatusDetails. Output seems very generic.